src/main/java/com/revature/domain/Eval.java

```diff
@@ -0,0 +1,126 @@
+package com.revature.domain;
+
+import java.io.Serializable;
+import java.sql.Date;
+import java.util.Objects;
+import javax.persistence.*;
+
+
+
+@Entity
+@Table(name="ie_eval")
+public class Eval implements Serializable{
+	
+	@Id
+	@Column(name="e_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="evalSeq")
+	@SequenceGenerator(allocationSize=1, name="evalSeq", sequenceName="eval_seq")
+	private Integer id;
+	
+	@Column(name="e_week")
+	private Integer week;
+	
+	@Column(name="e_date")
+	private Date date;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="e_eval_type")
+	private EvalType evalType;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="e_pid_trainee")
+	private Person trainee;
+	
+	@ManyToOne(fetch = FetchType.EAGER)
+	@JoinColumn(name="e_batch")
+	private Batch batch;
+
+	public Eval() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)